lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -225,6 +225,7 @@ public static function customerListFunction(array $params, $template)
         $customername = !isset($params['customername']) || $params['customername'];
 
         $form = isset($params['form']) ? $params['form'] : null;
+        $firstoption = $params['firstoption'] ?? 'Select customer';
```

Comment:
Po co akurat w tym jednym miejscu używasz skrótu operatorem `??` dostępnym od PHP 7.0?

Reply:
Czy to problem skoro wymagamy PHP7.1 w masterze?

Reply:
A nawet już 7.2 ;-)

Reply:
Zobacz jak dzięki temu znacząco uprościłem kod.